### ISSUE TITLE
How to set ace-editor content when container block is hidden

### ISSUE THREAD

**Issue by nodeofcode** [ajaxorg/ace#2497]
I am trying to set ace editor content when container block is hidden.

I am not able to to the same.

Here is what i am trying http://jsfiddle.net/U5JtP/408/

Why ace editor did not updated the content and how to update in such scenario?

**Comment by nightwing** [comment]
1. use newest version of ace: 1.1.9
2. call editor.resize() after showing the panel
see http://jsfiddle.net/U5JtP/409/

**Comment by nodeofcode** [comment]
Is there any internal configuration/patch, that doesn't require me to resize manually in the code.?

**Comment by nightwing** [comment]
No. It is not possible to detect when the size of the dom node changes.
So every time size of the container is modified it is needed to call `editor.resize` to notify ace about it.
